checkout.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial=scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <title>Checkout</title>
+  </head>
+  <body>
+    <header>
+      <a href="index.html">
+        <img src="images/adoptadog-logo.svg" alt="Adopt a Dog" width="300">
+      </a>
+      <h1>Seattle's premier dog adoption agency</h1>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+      <img src="images/cart-icon.svg" alt="cart icon" width="20">
+      <button type="button" name="button">Checkout</button>
+    </header>
+    <main>
+      <h2>Checkout</h2>
+      <form action="/action.php" method="GET">
```

Comment:
eventually we'll be using JS to do some work on this, so the php  tag _MIGHT_ confuse things. Probably not, but this might work as well .` <form action="" method="post" class="">` For now it's fine, but when we get to our basic form handling later it could cause wierdness